eng/testing/linker/trimmingTests.targets

```diff
@@ -166,8 +166,14 @@
 
     <Error Condition="'$(ExecutionExitCode)' != '100'" Text="Error: [Failed Test]: %(TestConsoleApps.ProjectCompileItems). The Command %(TestConsoleApps.TestCommand) return a non-success exit code $(ExecutionExitCode)." ContinueOnError="ErrorAndContinue" />
  
-    <!-- remove test project dir to save disk space if the test was successful -->
-    <RemoveDir Condition="'$(ExecutionExitCode)' == '100'" Directories="%(TestConsoleApps.ProjectDir)"  />
+    <!-- Remove test projects dir to save disk space if the test was successful. Ignore failures as this is best effort.
+         Don't use Removedir as ContinueOnError on it doesn't work when using warnaserror/TreatWarningsAsErrors. -->
```

Comment:
Any idea why RemoveDir doesn't work as ContinueOnError?

Reply:
because we set warnAsError=true so it would be an error again...

we use the same code in the VMR, Viktor shared it with me

Reply:
Got it. Not questioning anything, I am just curious.